improver/calibration/rainforest_calibration.py

```diff
@@ -126,11 +134,56 @@ def __new__(
             )
         return super(ApplyRainForestsCalibration, cls).__new__(cls)
 
-    def process(self) -> None:
-        """Subclasses should override this function."""
-        raise NotImplementedError(
-            "Process function must be called via subclass method."
-        )
+    def _check_num_features(self, features: CubeList) -> None:
+        """Check that the correct number of features has been passed into the model.
+        Args:
+            features:
+                Cubelist containing feature variables.
+        """
+        expected_num_features = self._get_num_features()
```

Comment:
Should this class function be defined here in the the Parent class?

Reply:
And should we test this?

Reply:
I've added the method in the base class. I don't think we need a test for it, since it doesn't do anything